### ISSUE TITLE
[Quest][Northrend] The last line of defense - Evading issue

### ISSUE THREAD

**Issue by thestranger064** [dalaranwow/dalaran-wow#5442]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Half the mobs are evading, frostbroods are evading and resetting.
Somehow they run around and resething their path, all the time. 
If you get them between resets on pathways you can kill the small monsters **Forgotten Depths Slayer.**

**Expected behaviour**: Once tagged they should be killable. A very important note is that according to
http://www.dalaran-wow.com/changelog/21/the-path-to-a-new-dawn -
The Last Line Of Defense: **Behaviour improvements to Frostbrood Destroyer.**

On live server this is not working at all, and what kind of fixes ? Once its on changelog, it should work.

https://github.com/dalaranwow/dalaran-wow/issues/5386 - Since its now still not working after **Live server update** this should stay reopened again.

**Steps to reproduce the problem**:

1.  Get the quest
2.  Get to the Rocket Launchers
3. Start attacking and see what happens.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
Screenshots of this happening

https://imgur.com/a/G35y3vd

**Comment by Lothloryen** [comment]
#5386
